lib/l10n/app_en.arb

```
@@ -655,6 +655,21 @@
         }
     },
 
+    "@_": {"description": "Task Budgets"},
```

RuboCop (1.73)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)